pytorch_tabular/config/config.py

```diff
@@ -513,7 +513,11 @@ def update_versions(self, name):
 class ModelConfig:
     """Base Model configuration
     Args:
-        task (str): Specify whether the problem is regression of classification.Choices are: regression classification
+        task (str): Specify whether the problem is regression of classification.Choices are: regression classification ssl
```

Comment:
comment can be changed to 
`
Specify whether the problem is regression, classification, or self supervised learning. Choices are: regression classification ssl
`

Reply:
Fixed